assets/scss/site.scss

```diff
@@ -4829,3 +4829,28 @@ a.user-icon:hover > span, a.user-icon:focus > span {
 .thumbnail-stack .caption .title {
     font-size: 13px;
 }
+
+/* ------------------------------------ =marketplace -- */
```

Comment:
This can go in `assets/scss/pages/_marketplace.scss` without the comment (import it after the other imports at the top). Mind the spaces after `:`, the semicolon on the `margin-left`, and the newline at the end of the file.